server/resolvers/article/queries.js

```diff
@@ -1,90 +1,29 @@
-      {
-        id: 6,
-        number: 6,
-        content:
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
-        keywords: ['Lorem', 'Ipsum'],
-        minutesToRead: 8
-      },
-      {
-        id: 7,
-        number: 7,
-        content:
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
-        keywords: ['Lorem', 'Ipsum'],
-        minutesToRead: 8
-      },
-      {
-        id: 8,
-        number: 8,
-        content:
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
-        keywords: ['Lorem', 'Ipsum'],
-        minutesToRead: 8
-      },
-      {
-        id: 9,
-        number: 9,
-        content:
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
-        keywords: ['Lorem', 'Ipsum'],
-        minutesToRead: 8
+  articles: async (_, { search }) => {
+    const payload = await searchApi(search)
+
+    if (!payload.error) {
+      const articleIds = [...payload.data.articles]
+      const articles = []
+
+      for (const id of articleIds) {
+        const article = await databaseApi(id)
+
+        if (article.error) return new Error(JSON.stringify(article))
+
+        article.minutesToRead = parseInt(article.wordCount) / AVERAGE_WORDS_PER_MINUTE
+        delete article.wordCount
+
+        articles.push(article)
       }
-    ]
+
+      return articles
+    }
+
+    return new Error(JSON.stringify(payload.error))
```

Comment:
nit: Consider inverting the `if (!payload.error)` condition so that it looks like

```javascript
if (payload.err) return new Error(JSON.stringify(payload.error))
```

I think the code will be easier to read that way.